wdio.conf.js

```diff
@@ -8,94 +8,121 @@ const { getScreenshotName, getRefPicName, getDiffScreenshotName } = visualRegres
 
 const resolution = { width: 1400, height: 1050 };
 const screenResolution = resolution.width.toString() + 'x' + resolution.height.toString();
-const browserName = 'chrome';
-const e2eProfile = process.env.E2EPROFILE || '';
+const e2eProfile = (process.env.E2EPROFILE || '').split(',');
```

Comment:
`const e2eProfile = (process.env.E2EPROFILE || '').split(',').filter(profile => profile !== '')`

so later you don't need to:
`if (e2eProfile[0] !== '')`

you can just do:

`if (e2eProfiles.length === 0) { // default `